configs/katello/4.13.yaml

```diff
@@ -0,0 +1,39 @@
+---
+:project: katello
+:github_org: katello
+:nightly: false
+:releases:
+  :4.13.0:
+    :redmine_version_id: 1788
+:prior-releases:
+  :4.12.0:
```

Comment:
Also need https://projects.theforeman.org/projects/katello/roadmap#Katello_4.12.1

Reply:
And https://projects.theforeman.org/projects/katello/roadmap#Katello_4.11.2 to be safe.

Reply:
@ianballou updated